src/main/resources/application-integration.properties

```diff
@@ -0,0 +1,42 @@
+transcription.api.url=http://fresenia.man.poznan.pl/tp-api
+server.servlet.context-path=
+europeana.api.tp.authorization-token=wh2B55drz8lGyTDnTYO0nT1Dr6FBKGR0
+# Enabling H2 Console
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+spring.datasource.url=jdbc:h2:~/test;
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.servlet.multipart.max-file-size=100MB
+spring.servlet.multipart.max-request-size=100MB
+spring.jpa.hibernate.naming-strategy=org.hibernate.cfg.ImprovedNamingStrategy
+# Keycloak integration
+keycloak.realm=EnrichEuropeana
+keycloak.auth-server-url=https://sso.apps.paas-dev.psnc.pl/auth
+keycloak.ssl-required=external
+keycloak.public-client=true
+keycloak.principal-attribute=preferred_username
+keycloak.use-resource-role-mappings:true
+keycloak.confidential-port=18080
+keycloak.resource=dei-frontend-app-dev
+# Europeana
+europeana.api.annotations.endpoint=http://annotation-api-acceptance.eanadev.org/annotation/
+europeana.api.record.endpoint=https://www.europeana.eu/api/v2/record
+#europeana.api.record.endpoint=http://metis-preview-api-prod.eanadev.org/api/v2/record
+europeana.search.api.url=https://www.europeana.eu/api/v2/search.json
+#europeana.search.api.url=http://metis-preview-api-prod.eanadev.org/api/v2/search.json
+api.clientId=transcribathon-dei
+api.clientSecret=59fc4002-a020-4abb-afc6-4be680f08034
+api.username=helin
+api.password=588eec26-0cb9-4a09-90b5-74365e1e7b43
+api.tokenEndpoint=https://auth.europeana.eu/auth/realms/europeana/protocol/openid-connect/token
+# Search API
+ddb.api.key=yVu8QOlMksagoWiamZfkuDDsAgEho6Ozm4haAks2FuzmMGWsXXb1558442839394
+ddb.api.url=https://api.deutsche-digitale-bibliothek.de
+ddb.search.api.url=/search
+ddb.format.api.url=/items/{recordId}/binaries
+ddb.binary.api.url=/binary/
+ddb.api.itemsEndpoint=/items/{recordId}/edm
+#conversion.iiif.server.url=iks-portal.synat.pcss.pl
+conversion.iiif.server.url=rhus-209.man.poznan.pl
```

Comment:
iiif server should be the test instance, not production (rhus-148)

Reply:
changed